extensions/amp-story/0.1/system-layer.js

```diff
@@ -212,15 +213,17 @@ export class SystemLayer {
     this.root_.addEventListener('click', e => {
       const target = dev().assertElement(e.target);
 
-      if (target.matches(
+      if (matches(
+          target,
```

Comment:
Minor: For readability, let's put this on the previous line and indent the following by +2 spaces.

```js
if (matches(target,
    `.${EXIT_FULLSCREEN_CLASS}, .${EXIT_FULLSCREEN_CLASS} *`)) {
```

Ditto for line 220 below.